v2/internal/frontend/desktop/windows/go-webview2/pkg/edge/ICoreWebView2Controller.go

```diff
@@ -130,3 +131,36 @@ func (i *ICoreWebView2Controller) NotifyParentWindowPositionChanged() error {
 	}
 	return nil
 }
+
+func Float64fromBits(b uint64) float64 {
```

Comment:
Would it be possible to use math.Float64frombits and math.Float64bits instead?

Reply:
Oh thanks, I did not know about them, they are now used